CMakeLists.txt

```diff
@@ -295,6 +296,13 @@ else()
   add_definitions(-DMXNET_USE_NCCL=0)
 endif()
 
+if (USE_INT64_TENSOR_SIZE)
```

Comment:
Can you also add this to the runtime feature detection code please?

https://mxnet.incubator.apache.org/api/python/libinfo/libinfo.html

Reply:
Added.